CreateDB

```diff
@@ -30,14 +32,15 @@ CREATE TABLE Messages (
     created_on TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP
 );
 
-CREATE TABLE Multimedia (
+CREATE TABLE Photo (
```

Comment:
Why the name change?

Reply:
Trying to keep it as similar to the ER as possible.

Reply:
He mentioned an example in class where a group named an entity as “users” in the ER and then created a table called “accounts”. Something that shouldn’t happen